README.md

```diff
@@ -6,6 +6,8 @@
 
 Playwright is a Node.js library to automate [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API. Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
 
+> Looking for **Playwright for .NET?** Head over to https://github.com/microsoft/playwright-dotnet for the latest info and release!
```

Comment:
```suggestion
> More comfortable in another programming language? Playwright is also available for [Java](https://github.com/microsoft/playwright-java), [.NET](https://github.com/microsoft/playwright-dotnet) and [Python](https://github.com/microsoft/playwright-python).
```

There are more flavours available to Playwright, and (despite my clear bias) I'd prefer not to single out .NET :-)

Reply:
See I wasn't even aware of the other ones ;-) -- as the primary repo, this is great just to provide a small, but helpful addition to the readme that I don't think takes away at all from the official site, etc.  Thanks!

Reply:
Is there a way to rephrase it as something like "playwright.dev is where you learn about Playwright"? From there, they can pick a language?

Reply:
I think you are underestimating the confusion currently caused and how developers look for information and especially in open source environments when repos exist.  This felt simple but not a hill I'm willing o die on to be honest.  I logged the request because I fell into the trap. I'm a .NET developer and all the documentation and messaging didn't point to me. There are links to the GH repo and from there a dead-end to me for a .NET dev.  If we don't want people to go there then maybe we should take off the prominent link to the repo that's all.

Reply:
Sorry about the hill feeling, not my intention here.

> If we don't want people to go there then maybe we should take off the prominent link to the repo that's all.

You mean hide https://github.com/microsoft/playwright-dotnet altogether? We were considering turning Playwright into monorepo, so that main repo would consume all the others. But we were not sure our branching model would handle it before we released all the ports. We might revisit it. But yes, we definitely want to make it feel 'internal'. Current [Getting Started](https://github.com/microsoft/playwright-dotnet#getting-started) section there is plain harmful - it completely hides the actual [Getting Started](https://playwright.dev/dotnet/docs/intro) guide.

Reply:
Let me fix it: https://github.com/microsoft/playwright-dotnet/pull/1551